tests/test_benchmark_special_features.py

```diff
@@ -94,19 +97,36 @@ def test_benchmark_feature_parameters_as_timeseries(self, margs):
         electricity_price = "electricity_price"
         soc_min = "soc_min"
 
+        assert (
+            data[ENERGY_CONSUMPTION]["demand_01"][TIMESERIES][3] == 0
+        ), f"The NaN value of the demand profile is not replaced by a 0 value."
         for k in range(0, len(csv_data[diesel_efficiency])):
             assert data[ENERGY_CONVERSION][diesel][EFFICIENCY][VALUE][
                 k
-            ] == pytest.approx(csv_data[diesel_efficiency][k], rel=1e-6)
+            ] == pytest.approx(
+                csv_data[diesel_efficiency][k], rel=1e-6
+            ), f"The diesel efficiency is not the one defined in the csv file as a timeseries."
```

Comment:
I am not exactly sure what is meant by "is not the one defined in the csv file as a timeseries.", could you elaborate?

Reply:
Parameters can be defined as scalars or timeseries. These ones are defined by timeseries in csv files. Will do it more properly.